dart-sample/build.gradle

```diff
@@ -1,49 +1,66 @@
-apply plugin: 'com.android.application'
+buildscript {
+    ext.versions = [
+            'minSdk': 14,
+            'compileSdk': 26,
 
-android {
-    compileSdkVersion versions.compileSdk
+            'supportLibrary': '25.3.0',
+            'androidPlugin': '3.0.0',
+            'androidTools': '26.0.0',
+            'butterknife' : '8.8.1',
+            'release': '3.0.0',
+    ]
 
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_7
-        targetCompatibility = JavaVersion.VERSION_1_7
-    }
+    ext.deps = [
+            android: [
+                    'runtime': 'com.google.android:android:4.1.1.4',
+                    'gradlePlugin': "com.android.tools.build:gradle:${versions.androidPlugin}",
+            ],
+            parceler: [
+                    'runtime': 'org.parceler:parceler-api:1.0.4',
+                    'compiler': 'org.parceler:parceler:1.0.4'
+            ],
+            dart: [
+                'runtime': "com.f2prateek.dart:dart:${versions.release}",
+                'compiler': "com.f2prateek.dart:dart-processor:${versions.release}"
+            ],
+            henson:[
+                'runtime': "com.f2prateek.dart:henson:${versions.release}",
+                'compiler': "com.f2prateek.dart:henson-processor:${versions.release}"
+            ],
+            butterknife:[
+                'runtime': "com.jakewharton:butterknife:${versions.butterknife}",
+                'compiler': "com.jakewharton:butterknife-compiler:${versions.butterknife}",
+                'annotations':"com.jakewharton:butterknife-annotations:${versions.butterknife}"
+            ],
+
+            junit: 'junit:junit:4.12',
+            fest: 'com.squareup:fest-android:1.0.7',
+            robolectric: 'org.robolectric:robolectric:2.4',
+    ]
 
-    defaultConfig {
-        applicationId 'com.f2prateek.dart.example'
-        minSdkVersion versions.minSdk
-        targetSdkVersion versions.compileSdk
-        versionCode 1
-        versionName '1.0.0'
+    repositories {
+        jcenter()
+        google()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
     }
 
-    lintOptions {
-        textReport true
-        textOutput 'stdout'
+    dependencies {
+        classpath deps.android.gradlePlugin
     }
 }
 
-dependencies {
-    implementation project(':dart-annotations')
-    implementation project(':dart')
-    implementation project(':henson')
-    implementation deps.parceler.runtime
-    implementation "com.jakewharton:butterknife:8.8.1"
-
-    annotationProcessor deps.parceler.compiler
-    annotationProcessor "com.jakewharton:butterknife-compiler:8.8.1"
-    annotationProcessor project(':dart-processor')
-    annotationProcessor project(':henson-processor')
-
-    testImplementation deps.junit
-}
+subprojects {
+    repositories {
+        jcenter()
+        google()
+        // TODO: Remove this
+        mavenLocal()
```

Comment:
Temporary to verify sample works with new D&H version. Should be removed before we merge the PR.